NDcPP_v2_2e.adoc

```diff
@@ -2118,32 +2179,118 @@ _The preferred method for verification is the Subject Alternative Name using DNS
+_The purpose of the explicit selection in the SFR is to prevent the TOE to provide an override authorization for situations other than specified in the selection (e.g. one or more certificates in the certification path have been revoked and this status is known to the TOE).  Override may be sought or granted at any time, though this typically occurs when an invalid certificate is presented during connection setup.  Override decisions may be stored and then consulted later, to permit connections using these otherwise-invalid certificates to establish trusted channels without user or administrator action._
 
 _If DTLS is selected in FTP_ITC then certificate validity is tested in accordance with testing performed for FIA_X509_EXT.1/Rev._
 
 _If DTLS is selected in FPT_ITT, then certificate validity is tested in accordance with testing performed for FIA_X509_EXT.1/ITT._
 
-*FCS_DTLSC_EXT.1.4* The TSF shall [selection: _not present the_ _Supported Elliptic Curves/Supported Groups Extension, present the Supported Elliptic Curves/Supported Groups Extension with the following curves/groups:_ [selection: _secp256r1, secp384r1, secp521r1, ffdhe2048, ffdhe3072, ffdhe4096, ffdhe6144, ffdhe8192_] _and no other curves/groups_] in the Client Hello.
+*FCS_DTLSC_EXT.1.4* The TSF shall [selection: _not present the Supported Groups Extension, present the Supported Groups Extension with the following curves/groups:_ [selection: _secp256r1, secp384r1, secp521r1, ffdhe2048, ffdhe3072, ffdhe4096, ffdhe6144, ffdhe8192_] _and no other curves/groups_] in the Client Hello.
+
+*_Application Note {counter:appnote_count}_*
+
+_The “Supported Elliptic Curves” extension was renamed to “Supported Groups” by RFC 7919; however, both terms refer to the same extension._
+
+_The following table provides the ST author with instructions for completing the selections._
+
+
+[cols="1,1,1"]
+|===
+| |_not present the Supported Groups Extension_|_present the Supported Groups Extension_
+|_DTLS 1.3 with or without DTLS 1.2 or DTLS 1.0_|_Shall not select_|_Shall select with any secp curves or any ffdhe groups – If DTLS 1.2 or DTLS 1.0 is supported, see the following rows for additional instructions._
+|_DTLS 1.2 or DTLS 1.0 with support for at least one ECDHE ciphersuite_|_Shall not select_|_Shall select – at least one secp curve shall be selected_
+|_DTLS 1.2 or DTLS 1.0 with support for at least one DHE ciphersuite_|_May select_ +
+
+_This extension is not required, because older TLS implementations did not negotiate FFC parameters (e.g. Group 14)._|_May select – at least one ffdhe group shall be selected_
+|===
+
+_Subsequent versions of this cPP will require support for the Supported Groups extension and prohibit use of legacy DH groups for key establishment._
```

Comment:
@plughy2 line 2218 should not be italicized according to the NDcPP proposal doc v13d.

Reply:
FCS_DTLSC_EXT.1.4: un-italicized the text where required.